qt-bugs@ issue : none (not yet)
Qt Software task ID : none (not yet)
bugs.kde.org number : 190485 (at least)
applied: no
author: Alexis Menard <alexis.menard@nokia.com>

This patch fix the regression introduced in 4.5.1 with SVG rendering.
The problem appear when a gradient reference an another gradient which is after
in the svg.

Please clear your plasma cache as well.

This patch is already in 4.5 branch so it will be in 4.5.2.

--- a/src/svg/qsvgstyle.cpp
+++ b/src/svg/qsvgstyle.cpp
@@ -808,6 +808,7 @@ void QSvgGradientStyle::resolveStops()
                     static_cast<QSvgGradientStyle*>(prop);
                 st->resolveStops();
                 m_gradient->setStops(st->qgradient()->stops());
+	        m_gradientStopsSet = st->gradientStopsSet();
             }
         }
         m_link = QString();
